foreman/foreman.spec

```diff
@@ -380,6 +382,8 @@ Requires: %{?scl_prefix}rubygem(autoprefixer-rails) >= 5.2
 Requires: %{?scl_prefix}rubygem(autoprefixer-rails) < 6.0
 Requires: %{?scl_prefix}rubygem(patternfly-sass) >= 2.7
 Requires: %{?scl_prefix}rubygem(patternfly-sass) < 3.0
+Requires: %{?scl_prefix}rubygem(sprockets) >= 3
```

Comment:
This is in the wrong place, it should now be in the main foreman RPM provides, not assets (https://github.com/theforeman/foreman/commit/24f70e14f1da00760b46ce2377db1147d849cf73).